app/controllers/Auth/LoginController.js

```diff
@@ -41,30 +40,27 @@ class LoginController {
      *        "user": null
      *     }
      */
+    constructor(userRepository) {
+        this.userRepository = userRepository;
+    }
+
     async invoke(request, response) {
         const { email, password } = request.body;
 
-        const user = await User.findOne({
-            where: { email },
-            attributes: [
-                'id',
-                'userName',
-                'firstName',
-                'lastName',
-                'email',
-                'password'
-            ]
-        });
+        const user = await this.userRepository.getByEmail(email);
 
         if (!user) {
             return response
                 .status(StatusCodes.UNAUTHORIZED)
                 .json({ auth: false, token: null });
         }
 
-        if (Auth.comparePasswords(password, user.getDataValue('password'))) {
+        const userPassword = await this.userRepository.getPassword(user.id);
```

Comment:
fetch user only only with password and if passoword is ok then fetch whole user object